### PR TITLE
fix: switch apollo graphql fetchPolicy to network-only

### PR DESCRIPTION
**Pull request by cbarber**
The default cache policy is resulting in queries cached on page refresh rather than querying the server. `network-only` will force new requests on page load and then rely on cache updates.

### CODE REVIEW

File: app/javascript/components/app.jsx [hunk 1]
```diff
@@ -58,6 +58,11 @@ persistCache({
 const client = new ApolloClient({
   link,
   cache,
+  defaultOptions: {
+    query: {
+      fetchPolicy: 'network-only',
```
Comment (geraldalewis) [inline app/javascript/components/app.jsx]: I would think `'cache-and-network'` would be preferable, no? This will never hit the cache...

**Comment by aaddaamm** [comment]
this needs a rebase @cbarber 🙏

File: app/javascript/components/app.jsx (hunk 1 above)
Comment (cbarber) [reply app/javascript/components/app.jsx]: https://www.apollographql.com/docs/react/api/react-apollo/#optionsfetchpolicy

Nope. Just first request. Then we're living the caching dream.

**Comment by cbarber** [comment]
@aaddaamm `git rebase -i adams-face`

**Comment by aaddaamm** [comment]
did that work? i'd be flattered if you renamed your local master to `adams-face`

**Comment by cbarber** [comment]
https://github.com/mojotech/helios2/pull/126

File: app/javascript/components/app.jsx (hunk 1 above)
Comment (chewnoill) [reply app/javascript/components/app.jsx]: agreed, `cache-and-network` seems like the play.

**Pull request merged**